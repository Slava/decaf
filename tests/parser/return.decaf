int f () {
  return 11 + 2;
}

