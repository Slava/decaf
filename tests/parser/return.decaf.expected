Program
  Function int: f:
    return (+ 11 2)
